api/services/content/v1/content.proto

```diff
@@ -75,6 +83,12 @@ message Info {
 
 	// CommittedAt provides the time at which the blob was committed.
 	google.protobuf.Timestamp committed_at = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
Should we rename this to `created_at` to follow the convention used across the api?